nixos/doc/manual/configuration/modularity.section.md

```diff
@@ -37,7 +37,7 @@ Note that both `configuration.nix` and `kde.nix` define the option
 [](#opt-environment.systemPackages). When multiple modules define an
 option, NixOS will try to *merge* the definitions. In the case of
 [](#opt-environment.systemPackages), that's easy: the lists of
-packages can simply be concatenated. The value in `configuration.nix` is
+packages can be concatenated. The value in `configuration.nix` is
```

Comment:
```suggestion
packages are just concatenated. The value in `configuration.nix` is
```

Reply:
I think it becomes less readable without "simply" or an equivalent.

Reply:
I don't know, still seems noisy to me. I tried making it more succinct, WDYT?

Reply:
Yes, better. The original wording would have made sense if it then showed another example of a type with complex merging rule, but like this is kind of pointless.